packages/custom-field/test/slot-wrapper.common.js

```diff
@@ -49,33 +48,13 @@ const fixtures = {
       <input type="number">
     </x-field2>
   `,
-  deep: `
```

Comment:
These tests fail with LitElement due to #6245, and also fail with Polymer when adding `await nextFrame()` to the `beforeEach()` block.